diff --git a/functions-common b/functions-common
index ff9bc0f..375ba7e 100644
--- a/functions-common
+++ b/functions-common
@@ -387,6 +387,8 @@ function GetDistro {
         DISTRO="xs${os_RELEASE%.*}"
     elif [[ "$os_VENDOR" =~ (kvmibm) ]]; then
         DISTRO="${os_VENDOR}${os_RELEASE::1}"
+    elif [[ "$os_VENDOR" =~ (Virtuozzo) ]]; then
+        DISTRO="vz$os_RELEASE"
     else
         die $LINENO "Unable to determine DISTRO"
     fi
@@ -420,7 +422,8 @@ function is_fedora {

     [ "$os_VENDOR" = "Fedora" ] || [ "$os_VENDOR" = "Red Hat" ] || \
         [ "$os_VENDOR" = "CentOS" ] || [ "$os_VENDOR" = "OracleLinux" ] || \
-        [ "$os_VENDOR" = "CloudLinux" ] || [ "$os_VENDOR" = "kvmibm" ]
+        [ "$os_VENDOR" = "CloudLinux" ] || [ "$os_VENDOR" = "kvmibm" ] || \
+        [ "$os_VENDOR" = "Virtuozzo" ]
 }
